apps/trading/client-pages/market/trade-grid.tsx

```diff
@@ -218,18 +223,12 @@ const MainGrid = memo(
 MainGrid.displayName = 'MainGrid';
 
 export const TradeGrid = ({ market, pinnedAssets }: TradeGridProps) => {
-  const wrapperClasses = classNames(
-    'h-full grid',
-    'grid-rows-[min-content_min-content_1fr]'
-  );
-
   return (
-    <div className={wrapperClasses}>
-      <MarketHeader />
+    <div className="h-full grid grid-rows-[min-content_1fr]">
       <div>
         <MarketBanner market={market} />
       </div>
-      <div className="min-h-0 p-0.5">
+      <div className="min-h-0 -mx-1 my-1">
```

Comment:
It'd be better to remove the x-padding from `TradeGridChild` rather than setting negative margin here.

Reply:
```suggestion
      <div className="min-h-0 mx-0 my-1">
```